app/lib/screens/app_onboarding/app_outdated_screen.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AppOutdatedScreen extends StatelessWidget {
+  static const String kPlayStoreUrl = 'https://play.google.com/store/apps/details?id=health.studyu.app';
+  static const String kAppstoreUrl = 'https://itunes.apple.com/app/id1571991198';
```

Comment:
Small thing, but for the sake of consistency let's start a `constants.dart` similar as the one for the Designer for the App.